data_import/views.py

```diff
@@ -48,3 +52,12 @@ def get_redirect_url(self, *args, **kwargs):
         access_log.save()
 
         return self.data_file.file_url_as_attachment
+
+
+class RemovedDataView(ListAPIView):
```

Comment:
Is this publicly accessible? This might not matter if this API isn't needed in the first place. I didn't think we would have another API view, but instead support a webhook/callback if provided.